Makefile

```diff
@@ -167,20 +173,23 @@ $(foreach target,$(DOCKER_TARGETS),$(eval $(call make-docker-target,$(target))))
 # test runs the test suite.
 test:
 	@echo "==> Testing ${NAME}"
-	@go test -timeout=30s -parallel=20 -tags="${GOTAGS}" ${GOFILES} ${TESTARGS}
+	@go test -timeout=30s -parallel=20 -tags="${GOTAGS}" ./... ${TESTARGS}
```

Comment:
This is probably fine for this tool. In Consul we have `$GOTEST_PKGS` which defaults to this but allows running only subpackage tests easily which is important in a large codebase. I don't know if it's really important here though as there are relatively few packages etc.

Reply:
Thanks for the reviews Paul. :)

My thought was that the `make test` call was primarily for automated testing (circleci, etc) and that if you were working on things and wanted to run specific tests you'd just use `go test` directly. 

Though I can see where that might come in handy if you are working with the same set of things over and over and want to just set them as environment variables. I'll consider that when I start working on the test suite to see if it would be handy.